seatunnel-e2e/seatunnel-connector-v2-e2e/connector-clickhouse-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/ClickhouseIT.java

```diff
@@ -76,7 +76,7 @@
 
 public class ClickhouseIT extends TestSuiteBase implements TestResource {
     private static final Logger LOG = LoggerFactory.getLogger(ClickhouseIT.class);
-    private static final String CLICKHOUSE_DOCKER_IMAGE = "yandex/clickhouse-server:22.1.3.7";
+    private static final String CLICKHOUSE_DOCKER_IMAGE = "clickhouse/clickhouse-server:latest";
```

Comment:
It's not a good idea to change the image to the latest version.

Reply:
@ruanwenjun  ClickhouseSinkCDCChangelogIT uses clickhouse/clickhouse-server:latest. Should we unify it to yandex/clickhouse-server:22.1.3.7